frontend/src/components/Profile/ProfileThreadsTab.js

```diff
@@ -0,0 +1,45 @@
+import {List, Space} from 'antd';
+import {ClockCircleOutlined, MessageOutlined} from "@ant-design/icons";
+import React from "react";
+import {Link} from "react-router-dom";
+import {format} from "date-fns";
+
+const ProfileThreadsTab = ({threads}) => {
+
+    const IconText = ({ icon, text }) => (
+        <Space>
+            {React.createElement(icon)}
+            {text}
+        </Space>
+    );
+
+    return (
+        <List
+            itemLayout="vertical"
+            size="large"
+            pagination={{
+                onChange: page => {
+                    console.log(page);
```

Comment:
console log :D Also, is something supposed to happen here?

Reply:
That's left over from ant design's template and it just logs what page you're on :P